torch/onnx/_internal/onnxruntime.py

```diff
@@ -0,0 +1,881 @@
+        #  GraphModule0
+        #   GraphModule1
+        #    node_0
+        #    node_1
+        #   node_2
+        #   GraphModule2
+        #    node_3
+        #    node_4
+        # by calling CapabilityBasedPartitioner.partition_and_fuse.
+        # Then, GraphModule1's and GraphModule2's forward method (GraphModule._wrapped_call)
+        # will be replaced by OrtBackend._ort_accelerated_call to delegate computation to ORT.
+        if graph_module in self._partitioner_cache:
+            partitioned_prim_graph_module = self._partitioner_cache[graph_module]
+        else:
+            prim_graph_module = graph_module
+            # TODO(wschin): this is required for removing aten::_to_copy in _replace_to_copy_with_to.
+            # See https://github.com/pytorch/pytorch/issues/106871.
+            _replace_to_copy_with_to(prim_graph_module)
+            partitioner = CapabilityBasedPartitioner(
+                prim_graph_module,
+                self._supported_ops,
+                allows_single_node_partition=True,
+            )
+            partitioned_prim_graph_module = partitioner.partition_and_fuse()
+            self._partitioner_cache[graph_module] = partitioned_prim_graph_module
+
+            # Overriding fused_module's __call__() function with ort_acclerated_call()
+            # This loop goes through all graph partitions (each of them is an ONNX-representable graph)
+            # and override their _wrappped_call function with _ort_accelerated_call.
+            # Inside _ort_accelerated_call, the partition's graph is exported into ONNX and executed by ORT.
+            for node in partitioned_prim_graph_module.graph.nodes:
+                # TODO(wschin): use a better way to identify fused submodule
+                # See https://github.com/pytorch/pytorch/issues/106872.
+                if node.op == "call_module" and "fused_" in node.name:
+                    fused_module = getattr(partitioned_prim_graph_module, node.name)
+                    # self.ort_acclerated_call is responsible for exporting graph to ONNX,
+                    # creating ORT session, and running ORT session.
+                    fused_module._wrapped_call = self._ort_acclerated_call
+
+        return partitioned_prim_graph_module
+
+    def __call__(
+        self, graph_module: torch.fx.GraphModule, args
+    ) -> torch.fx.GraphModule:
+        """Interface for dynamo to compile a graph module.
+
+        All backends/compilers are passed into dynamo as
+        callables following this signature (mapping from
+        (torch.fx.GraphModule, args) to torch.fx.GraphModule).
+        """
+        return self.compile(graph_module, args)
+
+
+def make_aot_ort(dynamic: bool = True):
+    """Wrap OrtBackend as PyTorch's AOT compiler.
+
+    Example usages:
+         # xdoctest: +REQUIRES(env:TORCH_DOCTEST_ONNX)
+         >>> import copy
+         >>> import torch
+         >>> from torch.onnx._backend.core import make_aot_ort
```

Comment:
This is now out of date due to the rename 😆 

```python
from torch.onnx._internal.onnxruntime import make_aot_ort
```

Reply:
Fixed and run well locally.